webrtc.html

```diff
@@ -8748,13 +8760,13 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
       </div>
       <div>
         <pre class="idl">dictionary RTCDataChannelInit {
-             boolean        ordered = true;
-             unsigned short maxPacketLifeTime;
-             unsigned short maxRetransmits;
-             USVString      protocol = "";
-             boolean        negotiated = false;
+             boolean         ordered = true;
+             unsigned short? maxPacketLifeTime = null;
+             unsigned short? maxRetransmits = null;
```

Comment:
Note that this makes `null` a valid input, i.e. users may specify `null`. Is that desirable? I think no.

The [advice](http://logs.glob.uno/?c=content#c437464) from our DOM team is *"'number or missing' is probably best represented as nullable, except in dictionaries/arguments where it should just be represented as optional"*

I think unless we want to allow users to pass in `null`, we're better off with non-nullable inputs, and some prose. Same for `id` below.